### DISCUSSION TITLE
Working on a internal install ps2 3.5v logic

### DISCUSSION THREAD

**Discussion by AceCuba** [darthcloud/BlueRetro#746] [Q&A]
So im working on a internal install with rst and pwr . i got a question the bluretro esp32 can suport logc3.5v , the pwr/rst line on the ps2 is alwys on higth state 3.5v , when bottom pres the line is sate to low 0v gnd . so my question is can i follow this schematics
![reset_sw](https://github.com/darthcloud/BlueRetro/assets/59297390/45cbac90-2ed8-4bd8-a97d-331c159d2340)
the 3.3v part

**Comment by Invictaz** [comment]
See Gusse on Twitter. He has the Methylene flexible pcb for internal Blueretro. It's even Blueretro HW2.
